client/nginx.go

```diff
@@ -910,6 +925,11 @@ func (client *NginxClient) GetStats() (*Stats, error) {
 		return nil, fmt.Errorf("failed to get stats %v", err)
 	}
 
+	slabs, err := client.getSlabs()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get stats %v", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to get stats: %v", err)
```

Just for consistency with the rest

Reply:
done